spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -47,25 +47,28 @@
 
   scenario "User deletes a step by step page", js: true do
     given_there_is_a_step_by_step_page_with_steps
-    and_I_visit_the_index_page
+    and_I_visit_the_publish_or_delete_page
     and_I_delete_the_step_by_step_page
     then_the_draft_is_discarded
     and_the_page_is_deleted
   end
 
   scenario "User publishes a page" do
     given_there_is_a_step_by_step_page_with_steps
+    and_I_visit_the_publish_or_delete_page
```

Comment:
Should this test have `when_I_view_the_step_by_step_page` as the step before?

Reply:
Done